.github/workflows/windows-msvc.yml

```diff
@@ -78,24 +76,24 @@ jobs:
             ${{ env.VCPKG_ROOT }}/packages
 
       - name: Bootstrap VCPKG
-        if: steps.restore-vcpkg.outputs.cache-hit != 'true'
```

Comment:
Do we want to bootstrap VPKG even if it was found in cache ?

Reply:
I _think_ that's needed to get the actual updates - several dependencies were out of date (and with the cache available that's super fast).

Reply:
Alright.